explanation/instances.md

```diff
@@ -66,12 +66,12 @@ An instance moves through different stages and correspondingly can have the foll
 | `prepared`        | AMS has decided the LXD node on which it will schedule the instance. |
 | `started`         | The instance is started and now booting. During the boot sequence, possible hooks are executed. Once all hooks have been executed, the instance will switch to `running`. |
 | `running`         | The instance is fully up and running. |
-| `stopped`         | The instance is fully stopped and will be deleted by AMS. |
+| `stopped`         | The instance is stopped. If you wish to restart the instance, use `amc stop <instance_id>`.|
```

Comment:
There is still a typo in this, it should be `amc start` to restart the instance. Either way should we even include this here ? why not just say that the instance is stopped and can be restarted if required...

Reply:
Oops, sorry. It should be fixed now.